test/TestRunner.py

```diff
@@ -70,6 +70,19 @@ def test_unicode_standard_color_formatting(self):
         formatter = ansiblelint.formatters.Formatter()
         formatter.format(matches[0], colored=True)
 
+    def test_runner_exclude_var_expansion(self):
+        filename = 'example/lots_of_warnings.yml'
+        os.environ['EXCLUDE_PATH'] = filename
```

Comment:
This test also needs a monkeypatch fixture.

Reply:
I'm not sure how you would like me to proceed here. I understand the need for the monkeypatch and that you are moving to pytest. But it uses the RulesCollection from the class to instantiate the Runner. To use the pytest monkeypatch I need to either pull this out of the class which would mean setting up Rules in the function, or setup pytest fixtures to decorate the class with, or I can use the unittest mocking features. How would you like me proceed.

Reply:
Went ahead an moved them out and just instantiated the rules in each test.